### DISCUSSION TITLE
screenshotting

### DISCUSSION THREAD

**Discussion by Oreolek** [leftwm/leftwm#648] [Q&A] [answered]
What's the config for screenshotting? I'm trying this:

```
[[keybind]]
command = "Execute"
value = "scrot -u"
key = "Print"
```

Is the key `Print` or `PrintScreen` or something else?

**Comment by VuiMuich** [comment, marked as answer]
`Print` should be ok, at least that's what the [keysym list](https://github.com/leftwm/leftwm/blob/main/leftwm-core/src/utils/xkeysym_lookup.rs#L46) says.
You can use `xev | awk -F'[ )]+' '/^KeyPress/ { a[NR+2] } NR in a { printf "%-3s %s\n", $5, $8 }'` to verify the key actually is actually executing the correct key-code

Edit: I see the problem now: you need to set a `modifier = []`, otherwise the parser breaks.

**Reply by Oreolek** [reply]
Thank you!
